src/java/org/apache/cassandra/utils/NoSpamLogger.java

```diff
@@ -84,21 +85,31 @@ private boolean shouldLog(long nowNanos)
             return nowNanos >= expected && compareAndSet(expected, nowNanos + minIntervalNanos);
         }
 
+        public boolean log(Level l, long nowNanos, Supplier<Object[]> objects)
+        {
+            if (!shouldLog(nowNanos)) return false;
+            return logNoCheck(l, objects.get());
+        }
+
         public boolean log(Level l, long nowNanos, Object... objects)
         {
             if (!shouldLog(nowNanos)) return false;
+            return logNoCheck(l, objects);
+        }
 
+        private boolean logNoCheck(Level l, Object... objects)
+        {
             switch (l)
             {
-            case INFO:
-                wrapped.info(statement, objects);
-                break;
-            case WARN:
-                wrapped.warn(statement, objects);
-                break;
-            case ERROR:
-                wrapped.error(statement, objects);
-                break;
+                case INFO:
+                    wrapped.info(statement, objects);
+                    break;
+                case WARN:
+                    wrapped.warn(statement, objects);
+                    break;
+                case ERROR:
+                    wrapped.error(statement, objects);
+                    break;
```

Comment:
Cassandra inherits most of Sun's java code style.. and for the switch statement, it seems no indentation for cases is required.
https://www.oracle.com/java/technologies/javase/codeconventions-statements.html#468

Reply:
Fair, although I'm just making this consistent with every other `switch` statement in the codebase.